module/applications/activity/activity-usage-dialog.mjs

```diff
@@ -130,7 +130,7 @@ export default class ActivityUsageDialog extends Application5e {
 
   /** @inheritDoc */
   async _prepareContext(options) {
-    // TODO: Adjust item scaling when scaling is changed
+    this.#item = this.#item.clone({ "flags.dnd5e.scaling": this.config.scaling ?? 0 });
```

Comment:
Is the item always cloned regardless of change to scaling? (re: #3724)

Reply:
Yes, though that occurs in the `use` method rather than here.